mobile/src/libs/components/icon-button/icon-button.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+import { TouchableOpacity } from '#libs/components/components';
+
+import { styles } from './styles';
+
+type Properties = {
+  onPress: () => void;
+  iconSourceSvg: JSX.Element;
+};
+
+const IconButton: React.FC<Properties> = ({ onPress, iconSourceSvg }) => {
+  return (
+    <TouchableOpacity onPress={onPress} style={styles.buttonContainer}>
+      {iconSourceSvg}
+    </TouchableOpacity>
+  );
+};
```

Comment:
Can we use only Button component which contain only icon without label?

Reply:
@v0ldemar01 thank you for noticing)